src/vision/detector.py

```
@@ -0,0 +1,677 @@
+        return np.expand_dims(rgb, axis=0).astype(np.uint8)
+
+    def _set_input(self, input_tensor: "NDArray[np.uint8]") -> None:
+        """Set the input tensor on the interpreter.
+
+        Args:
+            input_tensor: Preprocessed input tensor.
+        """
+        input_index = self._input_details[0]["index"]
+        self._interpreter.set_tensor(input_index, input_tensor)
+
+    def _parse_output(self, frame_width: int, frame_height: int) -> list[Detection]:
```

Ruff (0.14.10)

Unused method argument: `frame_width`

(ARG002)